shesha/supervisor/saxoPlusManager.py

```diff
@@ -38,22 +38,24 @@
 Initialization and execution of a saxo+ manager. 
 It instanciates in one process two compass simulations: 
 1 for first stage and 1 for the second stage (as defined in their relative .par files)
+The frequency ratio between the 2 stages (second stage is assumed to have higher frequency) must be specified, in order to properly call for each stage next method for their respective loops (accumulated integration on the first stage WFS and low frequency update of the first stage control).
 
 IMPORTANT:
 The next method of this manager --superseeds-- the compass next method so that the loop is fully handled by the saxoPlus manager. 
 
 Usage:
-  saxoPlusmanager.py <saxoparameters_filename> <saxoPlusparameters_filename> [options]
+  saxoPlusmanager.py <saxoparameters_filename> <saxoPlusparameters_filename> <frequency_ratio> [options]
```

Comment:
Could be a smoother change and easier to use if put as optionnal with a default value equal to 3 ?

Reply:
Addressed by commit [d484b28](https://github.com/ANR-COMPASS/shesha/pull/44/commits/d484b284b183f9b2c04c235e5f67392db45019de)